backend/src/main/scala/cromwell/backend/standard/callcaching/StandardCacheHitCopyingActor.scala

```diff
@@ -141,7 +141,7 @@ abstract class StandardCacheHitCopyingActor(val standardParams: StandardCacheHit
   when(Idle) {
     case Event(command: CopyOutputsCommand, None) if isSourceBlacklisted(command) =>
       // We don't want to log this because bucket blacklisting is a common and expected occurrence.
```

Comment:
Are these loggable/metricable comments in this file still relevant or should they be reworded?

Reply:
I believe their relevance is the same with these changes.